app/services/preserved_object_handler.rb

```diff
@@ -211,14 +247,16 @@ def with_active_record_transaction_and_rescue
   end
 
   # expects @incoming_version to be numeric
-  def update_preserved_copy(pres_copy, new_version, new_size)
+  def update_preserved_copy_version_etc(pres_copy, new_version, new_size, validated=false)
     pres_copy.version = new_version
     pres_copy.size = new_size if new_size
+    update_pc_validation_timestamps(pres_copy) if validated
   end
 
-  # expects @incoming_version to be numeric
-  def update_preserved_object(pres_obj, new_version)
```

Comment:
note that I decided the `update_preserved_object` one liner method was stupid and better served with inline code.